pkg/backends/tcp_test.go

```diff
@@ -7,11 +7,122 @@ import (
 	"reflect"
 	"sync"
 	"testing"
+	"time"
 
 	"github.com/ansible/receptor/pkg/logger"
-	"github.com/ansible/receptor/pkg/netceptor"
 )
 
+func TestTCPDialer_GetAddr(t *testing.T) {
```

Comment:
Consider an easy win to validate Preload and increase code coverage.

```suggestion
func TestTCPDialerCfg_PreReload(t *testing.T) {
    // Valid config - should pass
    cfg := TCPDialerCfg{Cost: 1.0}
    if err := cfg.PreReload(); err != nil {
        t.Errorf("PreReload() error = %v", err)
    }
    
    // Invalid config - should fail (proves it calls Prepare)
    cfg.Cost = -1.0
    if err := cfg.PreReload(); err == nil {
        t.Error("PreReload() expected error for negative cost")
    }
}

func TestTCPListenerCfg_PreReload(t *testing.T) {
    cfg := TCPListenerCfg{Cost: 1.0}
    if err := cfg.PreReload(); err != nil {
        t.Errorf("PreReload() error = %v", err)
    }
    
    cfg.Cost = 0.0
    if err := cfg.PreReload(); err == nil {
        t.Error("PreReload() expected error for zero cost")
    }
}

func TestTCPDialer_GetAddr(t *testing.T) {
```

Reply:
`PreReload` is wrapping `Prepare`, and I wrote tests for `Prepare`. That is why I skipped it :thinking: